THANKS.txt

```diff
@@ -171,3 +171,4 @@ Agilent which gave a genereous donation for support of SciPy.
 UC Berkeley for providing travel money and hosting numerous sprints.
 The University of Stellenbosch for funding the development of
     the SciKits portal.
+Google Inc.
```

Comment:
I made the THANKS line as specific as possible, referencing only this PR.  Hopefully that's OK!